### DISCUSSION TITLE
Share with other user?

### DISCUSSION THREAD

**Discussion by Michael2222** [te-online/timemanager#105] [General]
Hallo,

it is possible, that I can share a time project to another user?
Currently only the owner see the project to add times to a project.

Regards
    Michael

**Comment by te-online** [comment]
Hey Michael 👋 

Right now you can only share a whole client (and all its projects and tasks) with other users. 

From the share dialog

> You automatically grant read-only access to projects and tasks by sharing.
>
> Users you share with can create time entries.
>
> You can see all time entries, while others can only see and edit their own time entries.

Does that answer you question? 😊 
I'm sorry if that's not the answer you were hoping for. It's how it currently works and how it will work for some time.

**Comment by Michael2222** [comment]
Hello,

yes thank you, this is what I mean!

A small improvement maybe: It would be even nicer if you could do the authorization at project level and not at client level, since you might work with different people on different projects for one and the same client.

Regards!
    Michael.
